lib/widgets/body/faq.dart

```diff
@@ -22,120 +17,148 @@ Star: As an athlete's support token, it is non-tradable for now. However, the wa
     'Do I need a wallet to swap in NYXS?':
         'Firstly, you will have to purchase our coins on various exchanges globally. We will also have our coins on major local exchanges for ease of purchase in each of your countries. After this purchase, you can send the coins to the wallet on our mobile application.',
   };
+
+  @override
+  State<FAQ> createState() => _FAQState();
+}
+
+class _FAQState extends State<FAQ> {
   final List<bool> itemVisible = [];
 
   @override
   void initState() {
     super.initState();
-    for (var i = 0; i < items.length; i++) {
+    for (var i = 0; i < widget.items.length; i++) {
       itemVisible.add(false);
     }
   }
 
-  void changeVisibility(int id, [bool visible = true]) {
+  void changeVisibility(int id) {
     setState(() {
-      itemVisible[id] = visible;
+      itemVisible[id] = !itemVisible[id];
     });
   }
 
-  Widget get divider {
-    return const Divider(
-      color: Color(0xff838383),
-      height: 8.0,
-      thickness: 1.0,
-      indent: 18.0,
-      endIndent: 18.0,
-    );
-  }
-
-  Widget getQuestion(String question) {
-    return Text(
-      question,
-      style: getDescriptionStyle(
-        fontWeight: FontWeight.w700,
-        color: const Color(0xffFAFAFA),
-      ),
-    );
-  }
+  @override
+  Widget build(BuildContext context) {
+    List<FAQItem> fAQItems = [];
```

Comment:
How about using `faqItems`? I referred to [Style guide for Flutter repo](https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words).

Reply:
I agree with Yeha. faqItems looks better.

Reply:
True! Thanks for the feedback!